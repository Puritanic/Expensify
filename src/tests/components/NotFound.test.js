import React from 'react';
import NotFound from '../../components/NotFoundPage';

/* eslint-disable no-undef */

it('should render NotFound page correctly', () => {
  const wrapper = shallow(<NotFound />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
