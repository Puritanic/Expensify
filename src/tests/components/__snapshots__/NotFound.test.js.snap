// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound page correctly 1`] = `
<div>
  <p>
    Page not found
  </p>
  <p>
    Please go back to 
    <Link
      replace={false}
      to="/"
    >
      home page
    </Link>
     and try again :)
     
  </p>
</div>
`;
